packages/eslint-tool/README.md

```diff
@@ -0,0 +1,56 @@
+## @dotcom-reliability-kit/eslint-tool
+
+A prototype linting tool, specifically focussed on enhancing code quality and proactively catching errors/bugs before they make it into production. This is not a replacement for the [@financial-times/eslint-config-next](https://github.com/Financial-Times/eslint-config-next), but it can be installed alongside it or as an extension in it's config. This module is part of [FT.com Reliability Kit](https://github.com/Financial-Times/dotcom-reliability-kit#readme).
+
+- [Usage](#usage)
+  - [Static Code Analysis](#static-code-analysis)
+- [Contributing](#contributing)
+- [License](#license)
+
+## Usage
+
+---
+
+Install @dotcom-reliability-kit/eslint-tool as a dependency:
+
+```
+npm install --save @dotcom-reliability-kit/eslint-tool
+```
+
+Reference the shared configuration in your repository's own .eslintrc.js like so:
+
+```
+module.exports = {
+extends: ['@dotcom-reliability-kit/eslint-tool']
+}
+```
+
+Avoid using .eslintrc file format (without a file extension), as [this has been depracated](https://eslint.org/docs/latest/use/configure/configuration-files#configuration-file-formats).
+
+You can find the docs for ESLint shareable configs [here](https://eslint.org/docs/latest/extend/shareable-configs).
+
+### Static Code Analysis
```

Comment:
**question:** I'm unsure how much we should document use of ESLint, because most of our teams use it via [Tool Kit's eslint plugin](https://github.com/Financial-Times/dotcom-tool-kit/tree/main/plugins/eslint). Thoughts on this? Maybe we just link out to documentation? Unsure

Reply:
Ah ok - yeah makes sense to not link various docs and better to have one reference that people go to that's relevant for our org. I'll remove this line to avoid confusion